### ISSUE TITLE
Alternating Gap between Tabs

### ISSUE THREAD

**Issue by CuriousApe2020** [black7375/Firefox-UI-Fix#440]
Is the gap between Tabs supposed to alternate between a small gap and no gap? And if so is there any way to change that?
![gapnogap](https://user-images.githubusercontent.com/75225919/180893133-7f513a0f-b8b6-4c51-a5ff-e0b8af8f0204.png)

**Comment by thunderstone135** [comment]
I think the code below is the cause 👇 
```
    .tabbrowser-tab:not([last-visible-tab]) {
      margin-inline-end: -0.5px !important;
    }
```
I changed it to -1px and I think it fixed the gaps.

**Comment by CuriousApe2020** [comment]
Thanks it works I also changed:

>       .tabbrowser-tab {
>         padding-inline: 1px !important;
>       }
to 1.5 to have an gap

**Comment by thunderstone135** [comment]
> Thanks it works I also changed:
> 
> > ```
> >   .tabbrowser-tab {
> >     padding-inline: 1px !important;
> >   }
> > ```
> 
> to 1.5px to have an gap

I see. I just changed mine to make tiny lines between each tab.

**Comment by black7375** [comment]
@thunderstone135 It seems to work well as intended.
I remember you advised you a some times.
I will patch it and add to `CREDIT` list. 
Thank you :100:

**Comment by black7375** [comment]
Try to use `v5.3.1` 

https://github.com/black7375/Firefox-UI-Fix/releases/tag/v5.3.1
